test/integration/features/story_health.feature

```diff
@@ -53,6 +62,20 @@ Feature:
         And executing "oc expose svc/httpd-ex" succeeds
         And with up to "2" retries with wait period of "60s" command "curl -kI http://httpd-ex-testproj.apps-crc.testing" output should contain "HTTP/1.1 200 OK"
 
+    @windows
```

Comment:
How come this is different than above scenario, should be clubbed together.

Reply:
@tsedmik I thought I put the review commit for it but looks like it was not published. What I want is if we are only checking the status of a URI using the curl then we should replace it with a function which can use `http` module https://golang.org/pkg/net/http/ and provide the response back.

Reply:
@praveenkumar I'll submit this as a separate PR (replace all `curl` calls with a function call that uses Go's `http` module). Thanks for the tip.

Reply:
Just a follow-up, the change away from `curl` is ready here: #577.

Reply:
@jsliacan @tsedmik since #577 already merged now can you update this PR ?